### PR TITLE
WRO-10782: Update pattern-ls2request samples to use Redux Toolkit

### PR DESCRIPTION
**Pull request by alexandrumorariu**
Enact-DCO-1.0-Signed-off-by: Alexandru Morariu alexandru.morariu@lgepartner.com

### Checklist

* [X] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [X] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Updated pattern-ls2request(ui) and pattern-ls2request-camera(sandstone) samples to use Redux Toolkit.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRO-10782

### CODE REVIEW

**Review by adrian-cocoara-lgp — APPROVED** [review]
LGTM

File: sandstone/pattern-ls2request-camera/src/store/index.js [hunk 1]
```diff
@@ -152,3 +156,9 @@ export const closeCamera = (handle) => dispatch => {
 			});
 	});
 };
+
+const index = configureStore({
```
Comment (MikyungKim) [inline sandstone/pattern-ls2request-camera/src/store/index.js:160]: Can we rename `index` to `store`, please?

File: sandstone/pattern-ls2request-camera/src/components/CameraView.js [hunk 2]
```diff
@@ -5,15 +5,15 @@ import VirtualList from '@enact/sandstone/VirtualList';
...
 
-import {closeCamera, getCameraIds, startCamera} from '../actions';
+import {closeCamera, getCameraIds, startCamera} from '../store';
 
 const CameraView = () => {
 	const videoRef = useRef(null);
 	const dispatch = useDispatch();
-	const cameraIds = useSelector((store) => store.cameraIds);
-	const cameraStatus = useSelector((store) => store.cameraStatus);
+	const cameraIds = useSelector((store) => store.deviceList);
```
Comment (MikyungKim) [inline sandstone/pattern-ls2request-camera/src/components/CameraView.js:15]: Let's keep the original name `cameraIds`.

File: sandstone/pattern-ls2request-camera/src/components/CameraView.js [hunk 3]
```diff
@@ -5,15 +5,15 @@ import VirtualList from '@enact/sandstone/VirtualList';
...
+	const cameraStatus = useSelector((store) => store.status);
```
Comment (MikyungKim) [inline sandstone/pattern-ls2request-camera/src/components/CameraView.js:16]: Here also. `cameraStatus`.

File: ui/pattern-ls2request/src/store/store.js [hunk 4]
```diff
@@ -1,13 +1,62 @@
...
+	// NOTE: This does not dispatch and will not update the store directly. It's expected to be updated by the subscription.
+	return new LS2Request().send({
+		service: 'luna://com.webos.service.settings/',
+		method: 'setSystemSettings',
+		parameters: params,
+		onSuccess: () => {}, // we expect the value will be updated via subscribed callback and do nothing here
+		onFailure: (res) => console.error(res)	// eslint-disable-line no-console
+	});
+};
+const initialState = {};
```
Comment (MikyungKim) [inline ui/pattern-ls2request/src/store/store.js:56]:
```suggestion

const initialState = {};
```

File: sandstone/pattern-ls2request-camera/src/components/CameraView.js (hunk 3 above)
Comment (alexandrumorariu) [reply sandstone/pattern-ls2request-camera/src/components/CameraView.js]: Done

File: sandstone/pattern-ls2request-camera/src/components/CameraView.js (hunk 2 above)
Comment (alexandrumorariu) [reply sandstone/pattern-ls2request-camera/src/components/CameraView.js]: Done.

File: sandstone/pattern-ls2request-camera/src/store/index.js (hunk 1 above)
Comment (alexandrumorariu) [reply sandstone/pattern-ls2request-camera/src/store/index.js]: Done.

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**